modules/gapi/src/backends/common/serialization.cpp

```diff
@@ -695,7 +695,7 @@ I::OStream& ByteMemoryOutStream::operator<< (char atom) {
     return *this;
 }
 I::OStream& ByteMemoryOutStream::operator<< (wchar_t atom) {
-    for (std::size_t i = 0; i < sizeof(wchar_t); ++i) {
+    for (std::size_t i = 0; i < 4; ++i) {
         m_storage.push_back(0xFF & (atom >> (i * 8)));
     }
```

Comment:
Instead, you could just cast it to uint32_t and reuse the existing serialization routine.

I'd also add a `static_assert(sizeof(uint32_t == wchar_t))` but this may be too strict (cc: @alalek )

Reply:
It was implemented exactly this way initially, I will revert the changes. Also assert on `==` won't work on windows (only `static_assert(sizeof(wchar_t) <= sizeof(uint32_t))`)

Reply:
Changed. @alalek please check